ims/einstein/iscsi/tgt.py

```diff
@@ -137,17 +147,23 @@ def remove_target(self, target_name):
             if target_name in targets:
                 os.remove(self.TGT_ISCSI_CONFIG + target_name + ".conf")
                 # tgtarglist = ["sudo", "-S","tgt-admin", "--execute"]
-                command = "tgt-admin -f --delete {0}".format(target_name)
-                proc = subprocess.Popen(command, shell=True,
-                                        stdout=subprocess.PIPE,
-                                        stderr=subprocess.PIPE)
-                proc.communicate()
-                if proc.returncode != 0:
-                    raise iscsi_exceptions.UpdateConfigFailedException(
-                        "Deleting target failed at deleting target file stage")
+                # command = "tgt-admin -f --delete {0}".format(target_name)
+                # proc = subprocess.Popen(command, shell=True,
+                #                         stdout=subprocess.PIPE,
+                #                         stderr=subprocess.PIPE)
+                # proc.communicate()
+                # if proc.returncode != 0:
+                #     raise iscsi_exceptions.UpdateConfigFailedException(
+                #         "Deleting target failed at deleting target file stage")
```

Comment:
This is temporary just in case we want to go back to the subprocess way of doing stuff.